src/front/wgsl/mod.rs

```diff
@@ -1142,10 +1142,12 @@ impl Parser {
                             .append(crate::Expression::LocalVariable(var_id));
                         context.lookup_ident.insert(name, expr_id);
                         match init {
-                            Init::Variable(value) => crate::Statement::Store {
-                                pointer: expr_id,
-                                value,
-                            },
+                            Init::Variable(value) | Init::Uniform(value) => {
```

Comment:
I believe the old code was correct. If it's `Init::Uniform`, then the initial value is already stored in `crete::LocalVariable::init`, so no `Statement::Store` is expecged

Reply:
Lets go in detail at Matrix about this subject, as this is a tricky one to answer.